app/grants/management/commands/sent_cart_reminder.py

```diff
@@ -0,0 +1,103 @@
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+from datetime import datetime
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.db.models import Max, F
+from django.utils import timezone
+
+from dashboard.utils import get_tx_status, has_tx_mined
+from grants.clr import predict_clr
+from grants.models import Contribution, Grant, CartActivity, Subscription
+from grants.views import clr_active, round_end, next_round_start
+from marketing.mails import warn_subscription_failed, remember_your_cart
+from townsquare.models import MatchRound
+
+
+class Command(BaseCommand):
+    help = 'Sent reminder to user who forgot its cart '
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--test',
+            default=False,
+            type=bool,
+            help="Only process and display the carts to being delivered"
+        )
+        parser.add_argument(
+            '--full-cart',
+            default=False,
+            type=bool,
+            help="Should the cart being delivered partially"
+        )
+        parser.add_argument(
+            '--hours',
+            type=int,
+            help="Should the cart being delivered partially"
+        )
+
+
+    def handle(self, *args, **options):
+        last_activity_by_user = CartActivity.objects.filter(latest=True, created_on__gt=next_round_start).exclude(metadata=[])
+        count = 0
+        if options.get('hours'):
+            hours = options.get('hours')
+        else:
+            hours = int((round_end - datetime.now()).total_seconds() / 3600)
+
+        for activity in last_activity_by_user:
+            print(activity)
+
+            # Check if this cart is still valid
+            no_checkout_grants = []
+            for grant_entry in activity.metadata:
+                subscription = Subscription.objects.filter(grant_id=grant_entry['grant_id'],
+                                                           contributor_profile=activity.profile,
+                                                           created_on__gt=activity.created_on,
+                                                           created_on__lte=round_end).first()
+
+                if not subscription:
+                    no_checkout_grants.append(grant_entry)
+
+            if options['full_cart']:
+                if len(no_checkout_grants) != len(activity.metadata):
+                    print(f' * Activity {activity.id}: The grants were partially contributed but no notification will be delivered')
+                    continue
+
+            cart_query = [f'{grant["grant_id"]};{grant.get("grant_donation_amount", 5)};{grant.get("token_local_id", 283)}'
```

Comment:
whats up with this 5 DAI default?

Reply:
The first version of share cart doesn't include the amounts, and the default amount when a grant is added to the cart is 5 DAI.